specification/includes/physical-properties.json

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "object",
+  "required": [
+    "weight"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "height": {
+      "type": "number",
+      "minimum": 0,
+      "example": 150,
+      "description": "Height in millimeters"
+    },
+    "width": {
+      "type": "number",
+      "minimum": 0,
+      "example": 300,
+      "description": "Width in millimeters"
+    },
+    "length": {
+      "type": "number",
+      "minimum": 0,
+      "example": 500,
+      "description": "Length in millimeters"
+    },
+    "volume": {
+      "type": "number",
+      "minimum": 0,
+      "example": 22.5,
+      "description": "Volume in litres"
+    },
+    "weight": {
+      "type": "number",
+      "minimum": 0,
```

Comment:
Shouldn't the minimum weight be 1? Doesn't a weight of 0 result in `null`?

Reply:
`0` and `null` are two different types. I'm using a minimum value of `0`, because we haven't discussed if we want the weight to be at least `1`. But a negative weight is always incorrect.